package.json

```diff
@@ -16,7 +16,7 @@
     "immer": "^8.0.0",
     "query-string": "^6.13.7",
     "react": "^17.0.1",
-    "react-big-scheduler": "^0.2.6",
+    "react-big-scheduler": "0.2.6",
```

Comment:
sorka za to 😏

Reply:
luzik, ta biblioteka kalendarza jest popaprana i działa u nas tylko w jednej konfiguracji wersji :p